Remove driver date from struct drm_driver

In Linux 6.14 the field 'date' has been removed from the struct drm_driver.
See commit cb2e1c2136f716 drm: remove driver date from struct drm_driver and all drivers

Adjusted the drm_driver structure initialization accordingly for kernels >= 6.14.
---
 module/evdi_drm_drv.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/module/evdi_drm_drv.c
+++ b/module/evdi_drm_drv.c
@@ -147,7 +147,9 @@
 
 	.name = DRIVER_NAME,
 	.desc = DRIVER_DESC,
+#if KERNEL_VERSION(6, 14, 0) > LINUX_VERSION_CODE
 	.date = DRIVER_DATE,
+#endif
 	.major = DRIVER_MAJOR,
 	.minor = DRIVER_MINOR,
 	.patchlevel = DRIVER_PATCH,
